packages/babel-traverse/src/hub.js

```diff
@@ -1,6 +1,14 @@
+import Scope from "./scope";
+export interface HubInterface {
+  mark?: (type: string, message: string) => void;
+  addHelper?: (name: string) => Object;
+  getScope?: () => Scope;
+  getCode?: () => string;
+  buildError:(node: Object, msg: string, Error: Error) => Error;
+}
+
 export default class Hub {
-  constructor(file, options) {
-    this.file = file;
-    this.options = options;
+  buildError(node, msg, Error): Error {
```

Comment:
I added a test for this, though the test doesn't seem very necessary. 
It did pass the coverage test now! Thanks for reviewing :)